### ISSUE TITLE
Continuous Prediction training

### ISSUE THREAD

**Issue by Gytis123** [OlafenwaMoses/ImageAI#582]
Hello

If I continue training from old model only with new images (after removing old). Can i continue training from model which has accuracy of 1.000000, but only with new images (i deleted all previous images) representing the same items?

**Comment by OlafenwaMoses** [comment]
@Gytis123  Yes. You can set `continue_from_model=path_to_old_model` in your `.trainModel()` function.
